src/main/scala/com/atomist/tree/pathexpression/Predicate.scala

```diff
@@ -128,14 +128,22 @@ case class PropertyValuePredicate(property: String, expectedValue: String) exten
                         ee: ExpressionEngine,
                         typeRegistry: TypeRegistry,
                         nodePreparer: Option[NodePreparer]): Boolean = {
-    val extracted = n.childrenNamed(property)
-    if (extracted.size == 1) {
-      val result = extracted.head.value.equals(expectedValue)
-      //println(s"Comparing property [$property] of [${extracted.head.value}] against expected [$expectedValue] gave $result")
-      result
+    if (property == "value") {
+      // Treat the value property specially
+      n.value == expectedValue
```

Comment:
`value` is a special property: The string value exposed by all nodes (which could potentially be built from their children, for container nodes). The special case is inelegant, and perhaps we should differentiate in the parser, but `value` legitimately is a special case.